app/controllers/users_controller.rb

```diff
@@ -494,10 +494,19 @@ def is_local_username
     usernames.each(&:downcase!)
 
     cannot_see = []
+    here_count = nil
+
     topic_id = params[:topic_id]
-    unless topic_id.blank?
```

Comment:
I think this check is still required so we do not execute a query when `topic_id` is blank